### ISSUE TITLE
RAG Retrieval Error

### ISSUE THREAD

**Issue by d-r-w** [lmstudio-ai/mlx-engine#4]
When uploading and sending a document, this error is eventually triggered:

```
Failed to send message

Fork path does not exist.

ENOENT: no such file or directory, access '/Applications/LM Studio.app/Contents/Resources/app/.webpack/main/retrievalworker.js'
```

```
ls -a /Applications/LM\ Studio.app/Contents/Resources/app/.webpack/main/ 
.                               embeddingworker.js              libsqlite-vector.dylib
..                              embeddingworker.js.LICENSE.txt  llmworker.js
810.js                          index.js                        llmworker.js.LICENSE.txt
810.js.LICENSE.txt              index.js.LICENSE.txt            native_modules
build                           liblmstudio                     watcher.node
bundled-models                  liblmstudioworker.js            worker.js
```

**Comment by yagil** [comment]
We've found the issue and we're fixing it. We'll publish the fix as 0.3.4 build 2. I'll share it here when it's ready

**Comment by yagil** [comment]
Should be fixed in 0.3.4 build 2, available in https://lmstudio.ai/download?os=mac
